container-images/mds-agency/package.json

```diff
@@ -10,19 +10,16 @@
   "scripts": {
     "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
     "image": "yarn bundle && ../../bin/mdsctl -c image-name=mds-agency:${npm_package_version} buildImage",
-    "test": "yarn test:eslint && yarn test:unit",
-    "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "exit 0"
+    "test": "yarn test:eslint",
+    "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'"
```

Comment:
We really ought to just be importing these scripts from a shared location if possible.

Reply:
I've been thinking that for a while as well. Maybe you can include with your `ts-node-dev` suggestion.

Reply:
+1 for `ts-node-dev`